cmd/deployctl/cmd/build.go

```diff
@@ -369,4 +399,7 @@ func init() {
 	collectCmd.Flags().Bool(NormalizeInterfaceMTUFilterArg, false, "Normalize interface MTU values")
 	collectCmd.Flags().Bool(NormalizeConsoleFilterArg, false, "Normalize serial console attributes")
 	collectCmd.Flags().Bool(MinimalConfigFilterArg, false, "Shorthand notation for adding all available filters")
+	collectCmd.Flags().Bool(AllowDRBDConfigArg, false, "To allow DRBD field from system instances")
+	collectCmd.Flags().Bool(AllowAllFileSystemsArg, false, "To allow all storage filesystems from system instances")
```

Comment:
"To allow all storage filesystems from system instances. false: deploy only backup and databse filesystems" would be. It is not mandatory but it would be better to show what config will be included when "false" is set for this parameter.

Reply:
edited please check